### PR TITLE
Aesthetic Improvements to Maps Page & internal refactoring to code

### PR DESCRIPTION
**Pull request by heatsink**
Aesthetic Improvements to Maps Page & internal refactoring to code

### CODE REVIEW

**Comment by heatsink** [comment]
screen shot 2018-05-24 at 8 46 39 pm

**Comment by AndiAndreas** [comment]
I Like! I will try the code if it looks good on phones with smal display. Then I will approve.

**Comment by AndiAndreas** [comment]
We may consider moving 
+  static const String KARLSTRASSE ='Karlstrasse Campus';
+  static const String LOTHSTRASSE ='Lothstrasse Campus';
+  static const String PASING  ='Pasing Campus';
to staticVariables file.

**Comment by heatsink** [comment]
Good call, thanks

File: lib/views/maps.dart [hunk 1]
```diff
@@ -118,136 +57,80 @@ class _MapPageState extends State<MapPage> {
...
+  _getCoordinates(String campus) {
+    switch (campus) {
+      case StaticVariables.KARLSTRASSE:
+        return karlLocation;
+      case StaticVariables.LOTHSTRASSE:
+        return lothLocation;
+      case StaticVariables.PASING:
+        return pasingLocation;
+      default:
+        // TODO: Decide if it makes sense to default to this
```
Comment (RedTo) [inline lib/views/maps.dart]: Makes sense ;-)

**Pull request merged**